server/util/testing/compression/compression.go

```diff
@@ -0,0 +1,94 @@
+package compression
```

Comment:
nit: Can you move this to `server/testutil/testcompression/testcompression.go`? We started using this convention for newer test utils (though there are a few older test utils that need to be migrated to that convention)

Reply:
Done